server/verify_test.go

```diff
@@ -1065,3 +1117,139 @@ func TestVerifyAttestationWithSevSnp(t *testing.T) {
 		})
 	}
 }
+
+func TestVerifyAttestationWithTdx(t *testing.T) {
+
+	pcr0 := uint32(0)
+	algorithms := []struct {
+		ID         uint16
+		DigestSize uint16
```

Comment:
you don't need DigestSize here. You can just call hash.Size(): https://pkg.go.dev/hash#Hash

Reply:
Done